pkg/build/builder/cmd/builder.go

```diff
@@ -50,6 +50,7 @@ func run(builderFactory factoryFunc) {
 				glog.Fatalf("Cannot setup secret file for accessing private repository: %v", err)
 			}
 			if sourceURL != nil {
+				build.Annotations[bld.OriginalSourceURLAnnotationKey] = build.Spec.Source.Git.URI
```

Comment:
Because when we use the proxy we are changing the build source URL to the local proxy, we still need to keep track of what the original URL was when saving the metadata for the image
https://github.com/openshift/origin/pull/5908/files#diff-23acc8ba296661971473ad53a2cc00ccR27